<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>65</y>
     <width>686</width>
     <height>469</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="tree" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sortingEnabled" >
        <bool>false</bool>
       </property>
       <property name="animated" >
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus" >
        <bool>true</bool>
       </property>
      </widget>
      <widget class="PBrowser" name="viewer" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <family>Bitstream Vera Sans</family>
        </font>
       </property>
       <property name="openExternalLinks" >
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>Settings</string>
    </property>
    <addaction name="actionConfigure" />
   </widget>
   <widget class="QMenu" name="menu_Post" >
    <property name="title" >
     <string>&amp;Post</string>
    </property>
    <addaction name="actionEditPost" />
    <addaction name="actionDelete" />
   </widget>
   <widget class="QMenu" name="menuBlog" >
    <property name="title" >
     <string>Blog</string>
    </property>
    <widget class="QMenu" name="menuRegenerate_HTML" >
     <property name="title" >
      <string>Regenerate HTML</string>
     </property>
     <addaction name="actionRegenerateNeeded" />
     <addaction name="actionRegenerateAll" />
    </widget>
    <widget class="QMenu" name="menuRender_Pages" >
     <property name="title" >
      <string>Render Pages</string>
     </property>
     <addaction name="actionRender_Blog" />
     <addaction name="actionRender_Full_Blog" />
    </widget>
    <addaction name="separator" />
    <addaction name="menuRender_Pages" />
    <addaction name="menuRegenerate_HTML" />
   </widget>
   <widget class="QMenu" name="menu_File" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>166</height>
     </rect>
    </property>
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuImport" >
     <property name="title" >
      <string>Import</string>
     </property>
     <addaction name="actionImport_Advogato" />
     <addaction name="actionImport_PyDS" />
    </widget>
    <addaction name="separator" />
    <addaction name="actionNewPost" />
    <addaction name="actionNew_Story" />
    <addaction name="separator" />
    <addaction name="menuImport" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionBartleBlog_Help" />
    <addaction name="actionAbout_BartleBlog" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Post" />
   <addaction name="menuSettings" />
   <addaction name="menuBlog" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>534</y>
     <width>686</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>27</y>
     <width>686</width>
     <height>38</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewPost" />
   <addaction name="actionEditPost" />
   <addaction name="actionRender_Blog" />
   <addaction name="actionConfigure" />
  </widget>
  <action name="actionEditPost" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../resources/icons.qrc" >
     <normaloff>:/icons/icons/edit.svg</normaloff>:/icons/icons/edit.svg</iconset>
   </property>
   <property name="text" >
    <string>Edit Post</string>
   </property>
  </action>
  <action name="actionRender_Blog" >
   <property name="icon" >
    <iconset resource="../../resources/icons.qrc" >
     <normaloff>:/icons/icons/webexport.svg</normaloff>:/icons/icons/webexport.svg</iconset>
   </property>
   <property name="text" >
    <string>Render Blog</string>
   </property>
  </action>
  <action name="actionNewPost" >
   <property name="icon" >
    <iconset resource="../../resources/icons.qrc" >
     <normaloff>:/icons/icons/filenew.svg</normaloff>:/icons/icons/filenew.svg</iconset>
   </property>
   <property name="text" >
    <string>New Post</string>
   </property>
  </action>
  <action name="actionConfigure" >
   <property name="icon" >
    <iconset resource="../../resources/icons.qrc" >
     <normaloff>:/icons/icons/configure.svg</normaloff>:/icons/icons/configure.svg</iconset>
   </property>
   <property name="text" >
    <string>Configure BartleBlog</string>
   </property>
  </action>
  <action name="actionImport_Advogato" >
   <property name="text" >
    <string>Import Advogato</string>
   </property>
  </action>
  <action name="actionImport_PyDS" >
   <property name="text" >
    <string>Import PyDS</string>
   </property>
  </action>
  <action name="actionRegenerateNeeded" >
   <property name="text" >
    <string>Where necessary</string>
   </property>
  </action>
  <action name="actionRegenerateAll" >
   <property name="text" >
    <string>Everywhere</string>
   </property>
  </action>
  <action name="actionDelete" >
   <property name="icon" >
    <iconset resource="../../resources/icons.qrc" >
     <normaloff>:/icons/icons/delete.svg</normaloff>:/icons/icons/delete.svg</iconset>
   </property>
   <property name="text" >
    <string>delete</string>
   </property>
  </action>
  <action name="actionRender_Full_Blog" >
   <property name="icon" >
    <iconset resource="../../resources/icons.qrc" >
     <normaloff>:/icons/icons/webexport.svg</normaloff>:/icons/icons/webexport.svg</iconset>
   </property>
   <property name="text" >
    <string>Render Full Blog</string>
   </property>
  </action>
  <action name="actionAbout_BartleBlog" >
   <property name="text" >
    <string>About BartleBlog</string>
   </property>
  </action>
  <action name="actionNew_Story" >
   <property name="icon" >
    <iconset resource="../../resources/icons.qrc" >
     <normaloff>:/icons/icons/filenew.svg</normaloff>:/icons/icons/filenew.svg</iconset>
   </property>
   <property name="text" >
    <string>New Story</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="../../resources/icons.qrc" >
     <normaloff>:/icons/icons/exit.svg</normaloff>:/icons/icons/exit.svg</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBartleBlog_Help" >
   <property name="icon" >
    <iconset resource="../../resources/icons.qrc" >
     <normaloff>:/icons/icons/help.svg</normaloff>:/icons/icons/help.svg</iconset>
   </property>
   <property name="text" >
    <string>BartleBlog Help</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PBrowser</class>
   <extends>QTextBrowser</extends>
   <header>pbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/icons.qrc" />
 </resources>
 <connections/>
</ui>
